sdks/python/apache_beam/io/hdfs_integration_test/hdfs_integration_test.sh

```diff
@@ -48,6 +48,10 @@ cd ${CONTEXT_DIR}
 # https://github.com/docker/compose/issues/5745#issuecomment-370031631
 docker network prune --force
 
+# [BEAM-7405] The gcloud docker credential helper overwrites the docker's config file, which will
+# prevent the docker build with corrent credentials.
+rm -f $HOME/.docker/config.json
```

Comment:
the $HOME reference is concerning to me, since someone could run this test on their local machine, and it may be a surprising side-effect that some configs on their machine were deleted as a result.

Reply:
Good point. Explicitly remove the config from /home/jenkins/...

Reply:
Isn't this file necessary for running other tests? Would deleting it break them?

Have you tried setting the `DOCKER_CONFIG` environment variable?